changelogs/unreleased/gh-4450-logger-sigsegv.md

```diff
@@ -0,0 +1,5 @@
+## bugfix/core
+
+*  Fix potential nil dereference and crash in case if
```

Comment:
in case of an active log rotation during the

Reply:
Thanks! Fixed